services/graphql/src/graphql/definitions/app-user.js

```diff
@@ -202,6 +202,7 @@ type AppUserAuthentication {
   user: AppUser!
   token: AppUserAuthToken!
   loginSource: String
+  additionalEventData: JSONObject
```

Comment:
Rename, alpha sort props. Add docblock for the field in the context of this type.